resource/i18n/en-US.csv

```diff
@@ -352,3 +352,4 @@
 "Description","Description"
 "Read reviews","Read reviews"
 "Leave me review","Leave me review"
+"or login to your account","or login to your account"
```

Comment:
Is there 'login to your account' translation and is it still used? We can remove it if its not used. Same with 'or'.

Reply:
You are right. I removed `login to your account` translation but `or` is still used in other places in `vsf-capybara`.